docs/siem/detections/building-block-rule.asciidoc

```diff
@@ -0,0 +1,28 @@
+[[building-block-rule]]
+[role="xpack"]
+== About building-block rules
+
+Create building-block rules when you do not want to see their generated alerts
+in the UI. This is useful when you want:
+
+* A record of low-risk alerts without producing noise in the Alerts table.
+* Rules that execute on the alert indices (`.siem-signals-<kibana space>-*`).
+You can then use building-block rules to create hidden alerts that act as a
+basis for an 'ordinary' rule to generate visible alerts.
```

Comment:
Perfect description @benskelker! 🙂  

I've been curious what to call non-building-block detection rules, and I think `ordinary` in this context fits the bill. 👍